corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -43,7 +48,34 @@ def post_case_blocks(case_blocks, form_extras=None, domain=None):
         **form_extras
     )
     response, xform, cases = sp.run()
-    return xform
+    return xform, cases
```

Comment:
Yeah I was a little surprised too, but I do remember bumping into this last time I refactored this function; all of the other calls to it don't use the return value.

Nice catch on `CaseFactory`. That's only used in tests (and consequently very well tested as well) so I feel fine making that change and relying on the tests. It's simpler and may have the side effect of speeding up the tests a tiny bit too (though I wouldn't count on it being noticeable).